lib/Drush/Boot/BaseBoot.php

```diff
@@ -62,6 +62,10 @@ function bootstrap_and_dispatch() {
           $command += $this->command_defaults();
           // Insure that we have bootstrapped to a high enough
           // phase for the command prior to enforcing requirements.
+          if ($command['command'] == 'updatedb-batch-process') {
```

Comment:
I haven't really reviewed this patch in context, so I'm not sure at this point what the best method for passing this information down to `bootstrap_drupal_configuration()` is.